app/src/main/res/values/strings.xml

```diff
@@ -67,5 +67,16 @@
 
     <!-- Custom settings -->
     <string name="preferences_filename">yacguide_settings</string>
-    <string name="tourbook_ordering_key">Tourenbuch chronologisch ordnen</string>
+
+    <string name="order_tourbook_chronologically">Begehungen chronologisch ordnen</string>
+
+    <string name="collect_rocks">Felsen sammeln</string>
+    <string name="count_summits">Gipfel</string>
+    <string name="count_massifs">Massive</string>
+    <string name="count_boulders">Blöcke</string>
+    <string name="count_caves">Höhlen</string>
+    <string name="count_unofficial_rocks">inoffizielle Felsen berücksichtigen</string>
```

Comment:
Maybe "mitzählen" is better than "berücksichtigen"?

Reply:
I would remove the word `berücksichtigen` in all check boxes. Additionally, I prefer capitalized words at the beginning of each check bock string, even if it is not a noun.